codegen/src/main/java/io/improbable/keanu/codegen/python/DocString.java

```diff
@@ -0,0 +1,39 @@
+package io.improbable.keanu.codegen.python;
+
+import java.util.Map;
+
+class DocString {
+    private String comment;
+    private Map<String, String> params;
+
+    DocString(String comment, Map<String, String> params) {
+        this.comment = comment;
+        this.params = params;
+    }
+
+    private boolean isEmpty() {
+        return comment.isEmpty() && params.size() == 0;
+    }
+
+    String getAsString() {
+        if (isEmpty()) {
+            return "";
+        }
+        if (params.size() == 0) {
+            return "\"\"\"\n" + comment + "\n\"\"\"\n";
+        }
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("\"\"\"\n    ");
```

Comment:
likewise, you can make `"\n    "` a static final 

(that's supposed to say `\n` followed by 4 spaces but github has formatted it with one space)